stdlib/gc.ml

```diff
@@ -64,31 +64,30 @@ external get_minor_free : unit -> int = "caml_get_minor_free"
 external eventlog_pause : unit -> unit = "caml_eventlog_pause"
 external eventlog_resume : unit -> unit = "caml_eventlog_resume"
 
-open Printf
-
 let print_stat c =
   let st = stat () in
-  fprintf c "minor_collections:      %d\n" st.minor_collections;
-  fprintf c "major_collections:      %d\n" st.major_collections;
-  fprintf c "compactions:            %d\n" st.compactions;
-  fprintf c "forced_major_collections: %d\n" st.forced_major_collections;
-  fprintf c "\n";
-  let l1 = String.length (sprintf "%.0f" st.minor_words) in
-  fprintf c "minor_words:    %*.0f\n" l1 st.minor_words;
-  fprintf c "promoted_words: %*.0f\n" l1 st.promoted_words;
-  fprintf c "major_words:    %*.0f\n" l1 st.major_words;
-  fprintf c "\n";
-  let l2 = String.length (sprintf "%d" st.top_heap_words) in
-  fprintf c "top_heap_words: %*d\n" l2 st.top_heap_words;
-  fprintf c "heap_words:     %*d\n" l2 st.heap_words;
-  fprintf c "live_words:     %*d\n" l2 st.live_words;
-  fprintf c "free_words:     %*d\n" l2 st.free_words;
-  fprintf c "largest_free:   %*d\n" l2 st.largest_free;
-  fprintf c "fragments:      %*d\n" l2 st.fragments;
-  fprintf c "\n";
-  fprintf c "live_blocks: %d\n" st.live_blocks;
-  fprintf c "free_blocks: %d\n" st.free_blocks;
-  fprintf c "heap_chunks: %d\n" st.heap_chunks
+  let print fmt = Printf.fprintf c (fmt ^^ "\n") in
```

Comment:
What's the point?  My mental image of the `^^` operator is that it's expensive, drags much code in, and not well tested.  All this to save a `\n` at the end of a handful of format strings?

Reply:
`^^` was suggested by @dra27 in https://github.com/ocaml/ocaml/pull/10858/files#r780508792.
I'm happy to revert if people prefer that. Note that review should probably happen in #10858

Reply:
No longer part of this PR